whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy

```diff
@@ -1623,7 +1619,7 @@ class Inet6numSpec extends BaseQueryUpdateSpec {
         ack.summary.assertSuccess(2, 0, 2, 0, 0)
         ack.summary.assertErrors(0, 0, 0, 0)
 
-        ack.countErrorWarnInfo(0, 2, 2)
+        ack.countErrorWarnInfo(0, 3, 2)
```

Comment:
I did remove the mnt-lower here so I am getting
***Warning: Changing "mnt-lower:" value requires administrative authorisation

because the main object has mnt-lower and I am updating it